groups/groups.yaml

```diff
@@ -810,6 +810,23 @@ groups:
       - hi@amycod.es
       - prignanov@vmware.com
 
+  - email-id: k8s-infra-staging-provider-azure@kubernetes.io
+    name: k8s-infra-staging-provider-azure
+    description: |-
+      ACL for staging cloud provider Azure
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
```

Comment:
```suggestion
```

@nilo19 -- this line still needs to be dropped.
ref: @cblecker's [review](https://github.com/kubernetes/k8s.io/pull/440/files#r341828055)